newIDE/app/src/SceneEditor/index.js

```diff
@@ -1365,7 +1365,13 @@ export default class SceneEditor extends React.Component<Props, State> {
     const { project, layout } = this.props;
 
     const object = getObjectByName(project, layout, instance.getObjectName());
-    return !!object && object.is3DObject();
+    return (
+      !!object &&
+      gd.MetadataProvider.getObjectMetadata(
```

Comment:
This seems to add complexity to this method (in terms of performance). Can you check its usage across the codebase?

Reply:
The only usage seems to be to display a bubble when an instance is hovered.

Reply:
Perfect